subsys/bluetooth/host/Kconfig

```diff
@@ -351,6 +351,22 @@ config BT_OOB_DATA_FIXED
 	  value. This option should only be enabled for debugging and should
 	  never be used in production.
 
+config BT_KEYS_OVERWRITE_OLDEST
+	bool "Overwrite oldest keys with new ones if key storage is full"
+	help
+	  With this option enabled, if a pairing attempt occurs and the key storage
+	  is full, then the oldest keys in storage will be removed to free space
+	  for the new pairing keys.
+
+config BT_KEYS_OVERWRITE_OLDEST_CNTR_IN_SETTINGS
+	bool "Owerwrite aging counter will be stored in settings explicitly"
+	depends on BT_SETTINGS && BT_KEYS_OVERWRITE_OLDEST
+	default y
+	help
+	  With this option enabled, aging counter will be stored in settnigs explicitly.
```

Comment:
If CONFIG_BT_KEYS_OVERWRITE_OLDEST_CNTR_IN_SETTINGS enabled (), counter is stored in settings(together with keys) and is incremented every time we allocate a storage. It will take ages to overflow it even if we attempt to pair once a second. If not stored in settings counters are reinitialized after every reboot, and overflow is even more unlikely.

Reply:
Yes. I agree that overflow is unlikely. 

I was just pointing out that "settings" was spelled wrong here.